### DISCUSSION TITLE
Many Questions from a NEWB!!!!

### DISCUSSION THREAD

**Discussion by JesusGod-Pope666-Info** [Purfview/whisper-standalone-win#331] [Q&A] [answered]
Is the -m large-v2 the very best version - was there not a XXL version that was better? Or how does that work out. 

Can i use the Linux version in Termux, and if so, how to put it in there and find it make it do what I want. slight different commands maybe?

I have this now in a bat file on Windows 7: `start faster-whisper-xll.exe input -l English -m large-v2 -o output`
Should work, I think.... Although it is still doing its thing, being a very slow hacked slim client so still waiting for things to be seen produced by it, but it does find 15 files as it should but it will clearly take some hours :P.

Can I translate things as well? and does it support French, and can I then translate that into English?

Some of these files are very bad quality, like radio and all, is there something better then the large-v2 like that XXL thing, and how to get it working?

I also tried this, https://www.youtube.com/watch?v=XtcSqvx2Yfo - although still need to see a txt output file or something from it, but got the send to sorted and seems to work. I know this slim device is not the way to go in regards of speed, but I hope my mobile can do it, that should be a lot more powerful - however I get it working on it. I have Mobox and hope I can get something working on it, or maybe the direct Linux version might be even better.

But I need some help :)

So if I want a file frem French or Spanish and the very best transcriber option and then translating it - what code would I need for a command?

Thanks, it is very interesting to see - yet so confused at it goes for the moment.

Oh by the way, doing it on mobile, is it easy to make it use the GPU? And I guess by testing I can see what is best.
And this XXL transcript thing, how do I activate it, if there is anything like that, as large-v2.

And i am sure others have asked some of the questions if not all - just help me and my brain also writing out things. And it gets a little more personnel. In any case, I need some help. It is moving forward but slowly, and then finding new options that might be useful.

I have these French and Spanish things, that I really would like to have translated, and that might be possible, and even make them into lyrics "easily". That would be very cool.

-- Darkijah

**Comment by JesusGod-Pope666-Info** [comment]
Just a newbie.... I don't know..... There must be something... anyway, I have it unpacked for now and fixed an input and output folder in Gnulix.

![Screenshot_2024-11-18_14-57-02](https://github.com/user-attachments/assets/400bff4d-a05d-4ab0-8af6-b7d8ed4e9de3)

**Reply by Purfview** [reply]
In Linux you need to add dot and slash, for example: `./whisper-faster-xxl test2.wav -l en -m large-v2 -v true`

**Reply by JesusGod-Pope666-Info** [reply]
Great, I will try it out.... apparently I missed a slash I think, BRB :) Tanks.

**Reply by JesusGod-Pope666-Info** [reply]
> In Linux you need to add dot and slash, for example: `./whisper-faster-xxl test2.wav -l en -m large-v2 -v true`

It is a alive!!!! :D Thanks!!!!
![Screenshot_2024-11-18_18-19-11](https://github.com/user-attachments/assets/83d8b175-d4d0-4dce-9500-f75164568536)

**Comment by Purfview** [comment, marked as answer]
>Is the -m large-v2 the very best version - was there not a XXL version that was better?

Imo large-v2 is best currently, non-xxl is deprecated, it won't be updated anymore, it was same thing with less features and didn't include all libs needed to run it.

>I have this now in a bat file on Windows 7...

Windows 7 is not supported anymore since `r193.1` version, don't waste your time running it there... you'll get error on the first fallback.

>Although it is still doing its thing, being a very slow...

On a CPU it's slow, try `medium` or `turbo` models, accuracy difference is not very big vs `large`, btw, turbo is not suitable for translation.

>Can I translate things as well? and does it support French, and can I then translate that into English?

`--language French --task translate` will translate French to English

>Oh by the way, doing it on mobile, is it easy to make it use the GPU? 

There is no "mobile" version and mobiles doesn't have CUDA GPU.

>Some of these files are very bad quality, like radio and all, is there something better then the large-v2 like that XXL thing, and how to get it working?

"XXL" refers to a bloated package and just to have a different name from previous packaging.
If your audio contains music ect then you can try `--ff_mdx_kim2` option, but it's slow on CPU too.

**Reply by JesusGod-Pope666-Info** [reply]
okay, thanks for the info - I am trying to translate something at the moment, did find out. Although it goes very very slow on my slim device. But it is doing something. But I don't have that many foreign videos anyway, but it does seem to be worth the time so I can finally read what is being said.

Okay so Large_V2

Do you know if I can run this package or install this package in Termux on my Android mobile. My mobile is most likely the fastest device I have at hand..... And further fine tuned and boosted as well after rooting.

I'm not sure how this worked, as I could not get the file from the Linux version to work - so not sure if it is running something else - but I do not recall installing anything else, but maybe I did while toying around with things. But this is what I got from my Linux laptop, my crapbar. It seemed to work. Or do something, but.... yea still a long way to getting things to the point I want it.

darkijah@Darkijah-SanDisk:~/downloads/Faster-Whisper-XXL_r192.3.1_linux$ faster-whisper "Alan lamont.mp4" > transcript.txt
config.json: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2.31k/2.31k [00:00<00:00, 13.4MB/s]
vocabulary.txt: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 460k/460k [00:00<00:00, 1.58MB/s]
tokenizer.json: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2.20M/2.20M [00:00<00:00, 4.12MB/s]
model.bin: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 145M/145M [00:05<00:00, 24.8MB/s]
[2024-11-18 17:33:43.848] [ctranslate2] [thread 56162] [warning] The compute type inferred from the saved model is float16, but the target device or backend do not support efficient float16 computation. The model weights have been automatically converted to use the float32 compute type instead.
darkijah@Darkijah-SanDisk:~/downloads/Faster-Whisper-XXL_r192.3.1_linux$ faster-whisper "1.mp3" > transcript2.txt
[2024-11-18 17:44:37.647] [ctranslate2] [thread 56862] [warning] The compute type inferred from the saved model is float16, but the target device or backend do not support efficient float16 computation. The model weights have been automatically converted to use the float32 compute type instead.

**Reply by Purfview** [reply]
>Do you know if I can run this package or install this package in Termux on my Android mobile.

Read the answer.

**Reply by JesusGod-Pope666-Info** [reply]
> > Do you know if I can run this package or install this package in Termux on my Android mobile.
> 
> Read the answer.

Yea but it still has CPU's to run from.... So I just wonder if I can run it from it still. Maybe..... If I can get the application into Termux it should work I would think.... Maybe... I don't know....
I have a Windows version I can try, but still moving files at the moment.
Now moving some files in Linux as I filled the main drive up apparently.

**Reply by Purfview** [reply]
But still `There is no "mobile" version`, so I don't know what you will run there, mobile CPUs are not "x86" CPUs.

**Reply by JesusGod-Pope666-Info** [reply]
> But still `There is no "mobile" version`, so I don't know what you will run there, mobile CPUs are not "x86" CPUs.

I understand there is no mobile version - but the Linux version I would think should be able to run in Termux? If so, that might be the best option I have of computing power. I will try the windows version as well into Mobox, just waiting for it to finish moving files, going to take some time, clearly..... And waiting, waiting.

**Reply by Purfview** [reply]
I dunno what those apps do and doubt that you'll get anything useful from them as faster-whisper is not a simple app.

**Comment by JesusGod-Pope666-Info** [comment]
./fwx input -l en -m large-v2 -o output -v true --task translate

Not sure what the -v is about. In any case.... moving forward.

**Reply by JesusGod-Pope666-Info** [reply]
@Purfview what does -v true mean and stand for?

**Reply by JesusGod-Pope666-Info** [reply]
what does -v true do in whisper
ChatGPT said:
ChatGPT
In Whisper, the -v true or --verbose true flag is typically used to enable verbose mode, which provides additional details about the transcription process in the terminal. Here's what it does:

Purpose of -v true
Verbose Output: When -v true (or --verbose true) is used, the program outputs detailed logs during the transcription process. This can include information such as:
The progress of the model as it processes the audio file.
Details about which parts of the audio are being transcribed.
Any errors or warnings encountered during the process.
When to Use It
Debugging: Use this flag if you want to troubleshoot issues with the transcription, such as identifying where the program might be slowing down or failing.
Monitoring Progress: For long audio files, verbose mode can help you monitor the progress in real-time.
Usage Example
bash
Copy code
whisper audio.mp3 --model large --verbose true
or equivalently:

bash
Copy code
whisper audio.mp3 -v true
This will run the Whisper transcription with additional logging output in the terminal.

Default Behavior
If you don't specify -v or --verbose, the program might run in a less chatty mode, showing only minimal logs or errors. If you're unsure whether verbose mode is needed, you can omit it for a cleaner terminal experience.

**Reply by Purfview** [reply]
> In Whisper, the -v true or --verbose true flag is typically used to enable verbose mode, which provides additional details about the transcription process in the terminal. 

Yes.

**Comment by JesusGod-Pope666-Info** [comment]
Things are happening.... I think.
![Screenshot_2024-11-18_18-41-38](https://github.com/user-attachments/assets/73d402b2-7cc3-42a9-84b0-e780c6f428de)

**Reply by JesusGod-Pope666-Info** [reply]
![Screenshot_2024-11-18_18-44-24](https://github.com/user-attachments/assets/2c980d8e-78cf-4ac8-96cb-9051b3f5a287)

**Reply by Purfview** [reply]
That looks slow (from model loading time), what is your CPU, you don't run it in virtual machine?

**Reply by JesusGod-Pope666-Info** [reply]
> That looks slow (from model loading time), what is your CPU, you don't run it in virtual machine?

It is my Crapbar......... It is slow. That is why I wanna do it on my mobile! Both my Slim client and Crapbar is..... well very shitty.

So I think the Mobile will do a lot better if I can get it to do it and get all the commands sorted out and such.

**Reply by Purfview** [reply]
>So I think the Mobile will do a lot better 

I wouldn't waste time on that, faster-whisper is not CD-MAN where you need to emulate few cpu cycles.

**Reply by JesusGod-Pope666-Info** [reply]
./fwx input -l fr -m large-v2 -o output -v true --task translate

Lets see if this works, going to sleep, so I will let it run over the night. Hopefully it will work.

There was something about subtitles, dunno.... I guess there is also programs for that. But overall, I have those French.... oh no, just lost power, slim client instantly died, oh, well, nice to have a laptop with battery.
Well..... oh well. The Laptop should be faster anyway.

**Reply by Purfview** [reply]
For subtitle formatting you can try `--sentence` or `--standard`, anyway in your case you want to get `json` output too, as then you can input that json file and get new srt formatting without slow transcription process.

**Comment by JesusGod-Pope666-Info** [comment]
![Screenshot_2024-11-18_21-35-23](https://github.com/user-attachments/assets/3dedec7f-7c53-4784-b8e6-fb1e5d158442)
Pretty cool.

**Reply by Purfview** [reply]
In the title you can see progress bar and speed, `5:47:32` is your ETA.

**Comment by JesusGod-Pope666-Info** [comment]
Warning: Word-level timestamps on translations may not be reliable.
@Purfview any way to fix this? To make it more reliable.

**Reply by Purfview** [reply]
>  any way to fix this? To make it more reliable.

Don't do translation with whisper. Do it by some other AI or whatever.

**Comment by JesusGod-Pope666-Info** [comment]
can you finish the list of concentrated commands like this for whisper:
--model -m large-v2
--language -l English
--output -o C:/folder
ChatGPT said:
ChatGPT
Here's a list of Whisper's concentrated commands and their shorthand options, where applicable:

General Commands for Whisper
bash
Copy code
--model, -m <model_name>              # Specify the model to use (e.g., "large-v2").
--language, -l <language>            # Set the language for transcription or translation.
--output, -o <output_path>           # Set the output directory for results.
--device, -d <device>                # Specify the device for computation (e.g., "cuda" or "cpu").
--task, -t <task>                    # Choose task: "transcribe" or "translate".
--output_format, -of <format>        # Set the output format (e.g., "txt", "srt", "vtt", or "json").
--verbose, -v <true/false>           # Enable verbose mode (default: false).
--temperature, -temp <value>         # Adjust sampling temperature (default: 0).
--beam_size, -b <size>               # Set beam search size (default: 5).
--best_of <N>                        # Use N samples for sampling decoding (default: 5).
--fp16 <true/false>                  # Use FP16 (half-precision) for inference (default: true).
--threads, -th <number>              # Set the number of CPU threads for computation.
--prompt, -p <text>                  # Provide an optional text prompt to guide transcription.
--max_length, -ml <value>            # Set the maximum segment length (default: 30 seconds).

**Reply by Purfview** [reply]
Use `faster-whisper-xxl.exe --help` to see all commands and shortings too.

**Reply by JesusGod-Pope666-Info** [reply]
> Use `faster-whisper-xxl.exe --help` to see all commands and shortings too.

Yea kinda looked into those help things and was so confused. Never been good at those.

**Reply by Purfview** [reply]
Python's argparse output is confusing at first look.

**Reply by JesusGod-Pope666-Info** [reply]
> Python's argparse output is confusing at first look.

At least those help things, have never been me to understand much off. The setup seems confusing. So I tried to make my own little list and tried to get ChatGPT to further make more of it.
In any case it seems I am going somewhere now, but still a lot to learn.

Sadly just some very slow computers I have at hand for things. I still wanna see if I can get it to work on Termux and see the result. There is something about it here: https://www.reddit.com/r/termux/comments/13s41z9/does_openai_whisper_works_on_termux/

**Comment by JesusGod-Pope666-Info** [comment]
Well I tried 2 different windows emulators on Android, and it clearly did not work.....  Mobox and Winlator. Well had to give it a go and try. I have seen some talk about getting it to work on Termux, something I might further look into. For now, I have put both rubbish devices on task, and see what have happened after sleep. Although on speed, the Windows 7 Slim Client seems to be pretty way behind..... So, the Crapbar seems to do a lot better, even if it sucks as well. But we are getting something for sure. But I still think the Mobile could to the best - but maybe not. Whatever the case, I guess one could put the computer into action before sleep and take it little by little.

**Reply by JesusGod-Pope666-Info** [reply]
![Screenshot_2024-11-18_22-35-08](https://github.com/user-attachments/assets/0ba324ee-1070-41e8-9ac0-c74468239f9e)
I had to restart it as far as I recall while the slim client kept running - and it is already 9% compared to the Slim client with only 1% done. So...... Clearly.....
Anyway interesting to find these tools.

**Comment by JesusGod-Pope666-Info** [comment]
Well things are working it seems, I am using the tiny.en that seems to do just fine and a lot faster on my slow computers :)
Does not seem I need to use the best of the best, it does pretty well on tiny.en for English at least.

**Reply by Purfview** [reply]
Use base model at least, tiny is just bad.
Those smaller models can work OK on a clear speech, till you encounter accents, mumbling ect..

**Reply by JesusGod-Pope666-Info** [reply]
> Use base model at least, tiny is just bad. Those smaller models can work OK on a clear speech, till you encounter accents, mumbling ect..

Oh, it seems to do pretty good at least for now, I am sure I will see and notice issues if anyone comes by - but I have not seen anything bad on it yet. Might come around, I do have the other bigger version as well I can use. base.en I think it is called.
But so far so good with Tiny, not bad.... Well, it gives me some practice and some experience. Of cause I am using it for minor things for now helping me search out things, not something I will use for anything beside that.

I do have some French and Spanish videos - I will use the large-v2 for that of cause. But hey, learning by experience and getting somewhere, using Whisper :)

**Reply by JesusGod-Pope666-Info** [reply]
![Screenshot_2024-11-20_18-37-25](https://github.com/user-attachments/assets/51f0be8a-b898-4181-b4aa-291e3ceda41e)

**Reply by JesusGod-Pope666-Info** [reply]
./fwx "input/Tome Friess 3" -o "output/Tom Friess 120 126" -v true -m tiny.en --output_format txt

**Reply by JesusGod-Pope666-Info** [reply]
tiny.en :
[33:44.930 --> 33:49.550]  He's not telling you who the antichrist is.
[33:50.730 --> 33:53.710]  Now, is he really worth all the love and affection?
[33:54.710 --> 33:59.730]  That people love to ravish upon their spiritual leaders?
[34:01.470 --> 34:03.070]  You've got a question.
[34:04.610 --> 34:06.630]  Who your pastor is serving?
[34:08.190 --> 34:09.470]  Is he serving Christ?
[34:10.410 --> 34:12.010]  He says he is.
  Processing segment at 33:25.090
[34:13.090 --> 34:15.670]  He prays so wonderfully.
[34:17.210 --> 34:21.610]  He leads the congregation that says wonderful song and praise.
[34:23.030 --> 34:25.350]  But he's not telling you a critical piece of history.
[34:26.610 --> 34:27.970]  That makes all the difference.
[34:29.530 --> 34:34.010]  Does your pastor preach that the antichrist is one of the ancient Roman Caesars?
[34:34.770 --> 34:39.130]  Or possibly even Gregory the great of the ancient Greek empire?
  Processing segment at 33:52.210
[34:40.410 --> 34:45.970]  Or is he teaching you that the antichrist is a single individual yet future
[34:45.970 --> 34:49.770]  never before yet seen upon the earth?
[34:51.410 --> 34:54.230]  Now, how precious is that expensive church?
[34:55.810 --> 35:00.030]  How precious is that beautiful voice that sings and leads the choir?
[35:02.750 --> 35:04.990]  It falls to the ground, doesn't it?
[35:05.910 --> 35:07.670]  The greatest show on earth.
  Processing segment at 34:19.290
[35:07.670 --> 35:10.970]  That's what every church is in this country.
[35:11.850 --> 35:18.890]  If they don't teach the truth, there's nothing but a show, a pretense.
[35:20.630 --> 35:29.210]  It appears to be Christian, but there's vital information missing.
[35:31.890 --> 35:36.770]  Critical to your understanding of how this world works.
  Processing segment at 34:46.830
[35:36.770 --> 35:42.210]  Under the God of this world, the papacy.
[35:43.790 --> 35:45.650]  Martin Luther said it right.
[35:46.270 --> 35:50.370]  Listen to what Martin Luther said, and I'll paraphrase as close as I can remember it.
[35:51.450 --> 35:53.410]  We just read it here on Inquisition Update.
[35:53.590 --> 36:02.130]  Martin Luther said, the papacy is simply the mask for Satan himself.
[36:03.810 --> 36:04.710]  That's what he said.

**Reply by JesusGod-Pope666-Info** [reply]
Not sure what is going on here, Temperature stuff.....

* No speech threshold is met (0.930126 > 0.600000)
  Processing segment at 24:38.760
* Compression ratio threshold is not met with temperature 0.0 (19.066667 > 2.400000)
* Log probability threshold is not met with temperature 0.2 (-2.240100 < -1.000000)
* No speech threshold is met (0.916168 > 0.600000)
  Processing segment at 25:08.760
[33:20.080 --> 33:35.330]  I guess America, whereas I have something here a lot of these two. And this comes from the mother
[33:38.360 --> 33:43.460]  and it's in response to a new story.
  Processing segment at 25:38.760
* Compression ratio threshold is not met with temperature 0.0 (12.460317 > 2.400000)
* Compression ratio threshold is not met with temperature 0.2 (12.460317 > 2.400000)
* Compression ratio threshold is not met with temperature 0.4 (12.460317 > 2.400000)
* Compression ratio threshold is not met with temperature 0.6 (25.018868 > 2.400000)
* Compression ratio threshold is not met with temperature 0.8 (20.562500 > 2.400000)
* Log probability threshold is not met with temperature 1.0 (-1.180682 < -1.000000)
[33:46.920 --> 33:57.560]  And there it is. This is in regards to the article, quote court protecting a bit of personal privacy
[33:57.560 --> 34:05.200]  I'm quoted written by William staff fire of the fighting the 14th of January 1999.
[34:06.220 --> 34:13.000]  Great story, I just want to rally to Supreme Court and is still, on the American people side at least
* Reset prompt. prompt_reset_on_temperature threshold is met 1.000000 > 0.500000
  Processing segment at 26:05.040
[34:13.000 --> 34:20.240]  We won one small victory, now more America is a wake up and realize how important that
[34:20.240 --> 34:28.440]  single piece of paper is the Constitution of the United States of America, and this article
[34:28.440 --> 34:35.260]  that was stated, Clinton policy has been in the opposite direction toward a national ID

**Reply by Purfview** [reply]
>Not sure what is going on here, Temperature stuff.....

That happens when model is not sure about transcription, so it retries with increased temperature (aka "fantasy") to avoid being stuck in bad hallucination loops. You will encounter this on small models more often than on bigger ones. you can disable such fallback feature with `-fallback None`

**Reply by JesusGod-Pope666-Info** [reply]
> > Not sure what is going on here, Temperature stuff.....
> 
> That happens when model is not sure about transcription, so it retries with increased temperature (aka "fantasy") to avoid being stuck in bad hallucination loops. You will encounter this on small models more often than on bigger ones. you can disable such fallback feature with `-fallback None`

So you probably lose some information when this happens?

By the way it is near finish with the first year, which is the last year of radio shows he made and tried this with the files that was done - seems to work :) :

darkijah@Darkijah-SanDisk:/media/darkijah/HD1.82TN/Faster-Whisper-XXL_r192.3.1_linux/output/en The Hour of the Time - BC 1993-2001 (All Broadcasts)/THOTT - Bill Cooper 2001 - HourOfTheTime.com$ grep -iE 'pope|jesuit|jesus|christianity|jesuits|papacy|antichrist|illuminati' *.txt

0101__ - 1858.txt:[17:28.010 --> 17:33.010]  They believe that Jesus was actually born probably in four B.C.
0101__ - 1858.txt:[17:44.530 --> 17:51.430]  The true date of the beginning of this millennium, if you're measuring from the birth of Jesus,
0101__ - 1858.txt:[21:23.750 --> 21:28.830]  the actual first day of the third millennium since the birth of Jesus.
0101__ - 1858.txt:[21:37.670 --> 21:40.830]  well, why are you measuring since the birth of Jesus?
0101__ - 1858.txt:[21:43.330 --> 21:48.030]  The whole world measures since the birth of Jesus.
0101__ - 1858.txt:[21:52.870 --> 21:57.130]  And that's because for time Jesus was the greatest influence
0101__ - 1858.txt:[23:26.370 --> 23:28.170]  Hey, Jesus didn't go to Harvard.
0101__ - 1859B.txt:[57:08.000 --> 57:10.620]  The illuminati things that they're in might.
0101__ - 1860.txt:[36:59.540 --> 37:01.860]  triumph of treason, a Pierre Pope,
0101__ - 1861B.txt:[54:30.610 --> 54:32.890]  Don't look for some white Jesus
0101__ - 1861B.txt:[54:35.810 --> 54:37.130]  who's not Jesus at all
0101__ - 1861B.txt:[55:08.110 --> 55:09.770]  as Jesus or your savior
0102__ - 1867B.txt:[13:25.780 --> 13:30.360]  You see, when the Illuminati says the nation of world of people who will not use their
0102__ - 1873A.txt:[10:36.080 --> 10:37.740]  Jesus said that.
0102__ - 1873A.txt:[11:04.000 --> 11:04.980]  His name is Jesus.
0102__ - 1873A.txt:[54:34.250 --> 54:41.450]  Jesus is for the second place, and they are all young.
0102__ - 1880A.txt:[15:01.380 --> 15:09.420]  and the inconceivable grandeur of the immense volume of planes illuminating the battlefield from the burning of Charles Town,
0103__ - 1882A.txt:[34:08.790 --> 34:10.870]  You've called Christianity a religion for losers.
0103__ - 1882A.txt:[35:40.500 --> 35:43.640]  And I apologize for saying Christianity
0104__ - 1882B.txt:[19:18.130 --> 19:23.610]  Jesus said, see, the truth and the truth will make you free. Therefore,
0104__ - 1884.txt:[52:48.410 --> 52:58.470]  church full of Christian Native Americans who had converted to Christianity and built a little village
0105__ - 1887A.txt:[41:39.210 --> 41:39.890]  than from Jesus.
0105__ - 1887A.txt:[41:40.390 --> 41:41.310]  Jesus never tried
0106__ - 1895.txt:[38:26.000 --> 38:26.920]  was a member of the Illuminati.
0106__ - 1895.txt:[39:52.880 --> 39:54.320]  that Jesus will return
0106__ - 1896.txt:[53:21.800 --> 53:23.460]  Our was called the Illuminati.
0107__ - 1898(1).txt:[14:05.240 --> 14:06.260]  And attack on Christianity.
0107__ - 1898.txt:[14:03.630 --> 14:07.610]  That is a Christian and attack on Christianity and nothing less.
0108__ - 1906.txt:[25:11.640 --> 25:11.980]  Jesus.
0108__ - 1906.txt:[32:04.790 --> 32:07.670]  Nobody, unless, you know, Jesus comes back.
0108__ - 1906.txt:[53:51.660 --> 53:52.540]  uh, Jesuit order.
0108__ - 1907B.txt:[33:15.920 --> 33:17.640]  I believe in God, I believe in Jesus.
0109__ - 1911-7.txt:[16:51.770 --> 16:54.030]  Jesus is bad enough.
0109__ - 1913.txt:[13:08.020 --> 13:13.420]  It's what enabled Jesus to go to his death on the cross.
0109__ - 1914.txt:[11:23.970 --> 11:26.430]  And that I would not be willing to be telling Christianity
0109__ - 1916.txt:[41:09.340 --> 41:12.900]  if you're a player in the Illuminati you're a secular humanist
0109__ - 1917(1).txt:[47:57.790 --> 47:58.370]  in Christianity.
0109__ - 1917(1).txt:[53:18.130 --> 53:20.430]  and they're collectively known as the Illuminati.
0109__ - 1917.txt:[47:57.790 --> 47:58.370]  in Christianity.
0109__ - 1917.txt:[53:18.130 --> 53:20.430]  and they're collectively known as the Illuminati.
0110__ - 1921.txt:[21:34.910 --> 21:42.230]  looked at their homes upon order of the Pope
0110__ - 1921.txt:[22:14.430 --> 22:15.910]  in the Pope's religion.
0110__ - 1924.txt:[29:08.200 --> 29:08.760]  Jesus
0111__ - 1925.txt:[50:38.170 --> 50:38.870]  of Jesus
0111__ - 1926.txt:[15:45.610 --> 15:51.230]  I am in contact with a lot of us brown water sailors from iCORE, Joseph Christianity.

**Comment by JesusGod-Pope666-Info** [comment]
What would be the best settings for subtitles? Like when doing transcribing it sometimes make just 1 word and sometimes long sentences. Is there some default good setting for subtitles so the text becomes more like standard lines one after another?

**Reply by Purfview** [reply]
"Best" is what is best for you.
Read about `--sentence`, `--standard`, `--max_line_width` & ect...

**Comment by JesusGod-Pope666-Info** [comment]
Can you do a bilingual, Like English Hebrew?
Maybe translate the Hebrew and the English in a video to Danish?

And thanks for the last info.

**Comment by JesusGod-Pope666-Info** [comment]
I was thinking can it do Hebrew with Nikuds? The dots and Titles for how to pronounce something in Hebrew?
And can you have it write out the Hebrew in Letters and translate to English at the same time for subtitles.
